packages/fxa-payments-server/src/routes/Subscriptions/Cancel/CancelSubscriptionPanel.test.tsx

```diff
@@ -253,7 +253,9 @@ describe('CancelSubscriptionPanel', () => {
         ].forEach((x) => bundle.addResource(new FluentResource(x)));
         const plan = findMockPlan('plan_daily');
         renderWithLocalizationProvider(
-          <LocalizationProvider l10n={new ReactLocalization([bundle])}>
+          <LocalizationProvider
+            l10n={new ReactLocalization([bundle], undefined, () => {})}
```

Comment:
Q: Per L34 above: `import { LocalizationProvider, ReactLocalization } from '@fluent/react';`

Not sure if we could create a wrapper/class for `ReactLocalization` that automatically sets the null/empty `reportError` which would make this diff minimal?

```js
import { LocalizationProvider, ReactLocalization as _ReactLocalization } from '@fluent/react';

function ReactLocalization(bundles) {
  return new _ReactLocalization(bundles, null, () => {}); 
}
```

... although something better than that since it'd probably need to be a new `class` so the `new` keyword still works.

Reply:
I updated it in CancelSubscriptionPanel.test.tsx as you suggested, but left it pretty manual in the other places.